### PR TITLE
Support multiple push branches

### LINKED ISSUE #46: Add support for multiple push branches
We are going to be following the [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) development model which means we'll need to be able to push our docker images for multiple branches such as `master` and `develop`. I'm willing to put up the PR for the work, just wondering if there might be any specifics behind why this shouldn't be supported

### PR DESCRIPTION
**Pull request by vmaggioli**
Resolves #46

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -14,8 +14,8 @@ inputs:
   tags:
     description: 'String with tags, separated by a space'
     required: true
-  push-branch:
-    description: 'Specifies branch to push'
+  push-branches:
```
Comment (JeroenKnoops) [inline action.yml:17]: Changing the argument from `push-branch` to `push-branches` will break the API.
Maybe we can add both, and mark `push-branch` deprecated.

**Review by JeroenKnoops — CHANGES_REQUESTED** [review]
If you want, I can give it a try to add `push-branch`..

File: action.yml (hunk 1 above)
Comment (JeroenKnoops) [reply action.yml:17]: I've gave it a try in https://github.com/philips-software/docker-ci-scripts/commit/524f4cb88f6aee1c84c9d3d06f9a9c439c320b24
Can you include that commit in this PR?

**Comment by vmaggioli** [comment]
Merged in your commit

**Review by JeroenKnoops — APPROVED** [review]

**Pull request merged**